app/assets/stylesheets/user/profiles.scss

```diff
@@ -0,0 +1,10 @@
+.user-profiles {
+  .image > img {
+    height: 200px;
+    width: 200px;
+  }
+
+  input#user_profile_photo {
```

Comment:
Avoid qualifying id selectors with an element.<br>Selector `user_profile_photo` should be written in lowercase with hyphens